recipes/foxes/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "foxes" %}
+{% set version = "0.2.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/FraunhoferIWES/foxes/archive/v{{ version }}.tar.gz
+  sha256: 6dea991230ecedd06196fc0749279aa85ea278209cc75ffb37109e2defb24db1
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7,<3.11
+    - pip
+  run:
+    - python >=3.7,<3.11
+    - matplotlib-base
+    - numpy
+    - xarray
+    - dask-core
+    - distributed
+    - scipy
+    - netcdf4
+    - plotly
+    - python-kaleido
+
+test:
+  imports:
+    - foxes
+  requires:
+    - pip
+    - pytest
+  source_files:
+    - tests
+  commands:
+    - pip check
+    - pytest tests
+
+about:
+  home: https://pypi.org/project/foxes/
+  summary: Farm Optimization and eXtended yield Evaluation Software
+  description: |
+    The software foxes is a modular wind farm simulation and wake
+    modelling toolbox which is based on engineering wake models. It
+    has many applications, for example wind farm optimization,
+    wind farm post-construction analysis, wake model studies, and
+    wind farm simulations invoking complex model chains.
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - schmidtjonas
```

Comment:
This doesn't match the account making the PR here (@SchmJo). Is this a different account of yours?

Reply:
That's me - and I only have one account at github. I guess it is what grayskull picked up from PyPi? Will change it

Reply:
It's picked from git: https://github.com/conda-incubator/grayskull/blob/ecf5b1f3b60b994bc087b34d1fd94cd680713d70/grayskull/base/github.py#L130